account/account.go

```diff
@@ -103,6 +103,18 @@ func GetAccount(ctx context.Context, cfg *config.Configuration, fetcher stored_r
 		return nil, errs
 	}
 
+	var ipv6Errs []error
+	ipv6Errs = account.Privacy.IPv6Config.Validate(ipv6Errs)
+	if len(ipv6Errs) > 0 {
+		account.Privacy.IPv6Config.AnonKeepBits = iputil.IPv6BitSize
+	}
+
+	var ipv4Errs []error
+	ipv4Errs = account.Privacy.IPv4Config.Validate(ipv4Errs)
+	if len(ipv4Errs) > 0 {
+		account.Privacy.IPv4Config.AnonKeepBits = iputil.IPv4BitSize
+	}
```

Comment:
Nitpick: Could simplify the syntax...

```
if errs := account.Privacy.IPv6Config.Validate(nil); len(errs) > 0 {
	account.Privacy.IPv6Config.AnonKeepBits = iputil.IPv6BitSize
}

if errs := account.Privacy.IPv4Config.Validate(nil); len(errs) > 0 {
	account.Privacy.IPv4Config.AnonKeepBits = iputil.IPv4BitSize
}
```

Reply:
These lines show as having test coverage, but there doesn't seem to an assertion on setting their default values. I made a type locally where `IPv4Config.AnonKeepBits = iputil.IPv6BitSize` which is setting the ipv6 value for the ipv4 default and this mistake wasn't caught by a test.

Reply:
Yes, this is tested in a general GetAccount test. 
I added extra flag to assert IP config for account id "invalid_acct_ipv6_ipv4". 
I also realized I set both ipv6 and ipv4 to max value, instead of default mask bits. I added 2 new constants: 
```
IPv4DefaultMaskingBitSize = 24
IPv6DefaultMaskingBitSize = 56
```
For the simplified syntax I added new variables for errors to show explicitly we don't want to add IP config validation errors to `errs`.

Reply:
And also tested this update end to end including test with publisher account id.